samples/next-ab-testing/package.json

```diff
@@ -10,8 +10,8 @@
     "serve": "http-server ./out"
   },
   "dependencies": {
-    "next": "path_to_fork",
-    "next-experiments": "^1.0.4-alpha",
+    "next": "https://registry.npmjs.org/@sheertex/next/-/next-9.4.5-canary.14.tgz",    
```

Comment:
Could we use `resolutions` for this instead? See https://classic.yarnpkg.com/en/docs/selective-version-resolutions/

Reply:
According to the link you posted, we can specify only module version in resolution. In our case I make"next" module points  to our forked version, but saving original name "next".